src/scripts/get-car.ts

```diff
@@ -0,0 +1,12 @@
+import { Car } from './get-cars';
+
+async function getCar(id: number): Promise<Car> {
+  const responce = await fetch(`http://localhost:3000/garage/${id}`, {
+    method: 'GET',
+  });
+  const data: Car = await responce.json();
+
+  return data;
```

Comment:
Local variable 'data' is redundant
```suggestion
  return await responce.json();
```

And I hope you understand it, **async functions always return promise**.